### DISCUSSION TITLE
Plain WYSIWYG field

### DISCUSSION THREAD

**Discussion by mhdaouas** [directus/directus#8579] [Archive]
Is it possible to add a plain WYSIWYG field? What I mean is an HTML element without decorations (like buttons to change text style, add links, etc. and no borders). It can be useful in case of embedding an iframe. Also is it possible to make the element take 100% of the details page width (by default all the element rows take half of the width)?

**Comment by rijkvanzanten** [comment]
You could use the "code" interface with HTML as it's language, or even a plain textarea. You can set the width of the fields through the data model settings

**Reply by mhdaouas** [reply]
The code interface doesn't interprete the code, it just displays the code as text. Is there an option to check to be able to achieve this (display a real iframe)?

**Reply by joselcvarela** [reply]
Hello @mhdaouas 
I found no results while searching for iframe on source, so I think the only way to get this working is by creating a new custom interface (https://docs.directus.io/guides/interfaces/#custom-interfaces)

**Reply by rijkvanzanten** [reply]
Ohh I see, you're looking for a place to paste in HTML, but still have the ability to preview. You could simply disable every toolbar option but the "Edit Source Code" one and use that? 

https://user-images.githubusercontent.com/9141017/136978179-b7f4c4b3-a18f-4273-89dd-d2bbc5fd14e8.mp4

**Reply by joselcvarela** [reply]
Sorry for the misleading @mhdaouas

**Comment by Pachat** [comment]
And if you want ti get rid of the toolbar, this CSS will do the job
```
#myfield  .tox-editor-header {
	display: none
}
```
